db/migrate/20160322162318_add_opened_to_consoles.rb

```diff
@@ -0,0 +1,5 @@
+class AddOpenedToConsoles < ActiveRecord::Migration[5.0]
+  def change
+    add_column :consoles, :opened, :boolean, :default => false
```

Comment:
Since this has never been merged, can you just collapse the two migrations together, to reduce the migration churn?